torchci/pages/minihud/[repoOwner]/[repoName]/[branch]/[[...page]].tsx

```diff
@@ -289,14 +469,31 @@ function CommitSummary({ row }: { row: RowData }) {
         row={row}
         numPending={pendingJobs.length}
         showRevert={failedJobs.length !== 0}
+        ttsAlert={ttsConcerning.length > 0}
       />
-      <FailedJobs failedJobs={failedJobs} />
+      {!showDurationInfo && <FailedJobs failedJobs={failedJobs} />}
+      {showDurationInfo && (
+        <DurationInfoElement
+          concerning={ttsConcerning}
+          notConcerning={ttsNotConcerning}
+          expandAllDurationInfo={expandAllDurationInfo}
+        />
+      )}
     </div>
   );
 }
 
 function MiniHud({ params }: { params: HudParams }) {
   const data = useHudData(params);
+
+  let paramsNextPage = { ...params };
+  paramsNextPage.page = params.per_page * params.page + 1;
+  paramsNextPage.per_page = 1;
```

Comment:
Can you explain what this means? Like on page 1 we are fetching 51 commits? I guess so this is so you can get the delta for the last commit? Can we just fetch 51 commits in the first query? Cause otherwise you're kinda waiting for 2 API calls which could cause additional re renders/round trips.

Reply:
yes so on page 1 i want to fetch 1-51, then on page 2 i want to fetch 51-101, so the second query is for the 51st commit or the 101st commit

im not sure if i can just fetch 51 commits each time because i think it would result in 1-51 for the first page and then 52-102 for the second page